tests/Tests.elm

```diff
@@ -344,7 +344,44 @@ getSuite =
             ]
 
 
+groupSuite =
+    let
+        groupWhile f xs =
+            NE.groupWhile f xs
+                |> NE.toList
+                |> List.map NE.toList
+
+        xs1 =
+            NE.Nonempty ( 0, 'a' ) [ ( 0, 'b' ), ( 1, 'c' ), ( 1, 'd' ) ]
+
+        xs2 =
+            NE.Nonempty 1 [ 2, 3, 2, 4, 1, 3, 2, 1 ]
+    in
+        describe "group"
+            [ test "" <|
```

Comment:
Please give this test and the next one a name. (This will be an error in the new version of elm-test.)